packages/slate-react/src/components/editable.tsx

```diff
@@ -114,6 +115,7 @@ export const Editable = (props: EditableProps) => {
     readOnly = false,
     renderElement,
     renderLeaf,
+    renderPlaceholder = DefaultPlaceholder,
```

Comment:
This shouldn't be just the component as default but instead a function that returns the component like:

```tsx
renderPlaceholder = (props) => <DefaultPlaceholder {...props} />
```